src/main/java/stringCalculator/Calculator.java

```diff
@@ -0,0 +1,31 @@
+package stringCalculator;
+
+
+public class Calculator {
+
+    private String[] parsedInput;
+    private Operator operator;
+
+    Calculator(String[] parsedInput, Operator operator) {
+        this.parsedInput = parsedInput;
+        this.operator = operator;
+    }
+
+    int calculate() {
+        int result = Integer.parseInt(parsedInput[0]);
+
+        for (int idx = 1; idx < parsedInput.length; idx += 2)
+            result = operator.calculateWithOperator(result, parsedInput[idx], Integer.parseInt(parsedInput[idx + 1]));
+
+        return result;
+    }
+
+    static Calculator Of(String input) {
+        Operator operator = new Operator();
+        InputParser inputParser = new InputParser(input, operator);
+        String[] parsedInput = inputParser.parseInput();
+
+        return new Calculator(parsedInput, operator);
+    }
```

Comment:
```suggestion
    private Calculator(String[] parsedInput, Operator operator) {
        this.parsedInput = parsedInput;
        this.operator = operator;
    }

    public static Calculator of(String[] parsedInput, Operator operator) {
        return new Calculator(parsedInput, operator);
    }

    public static Calculator of(String input) {
        Operator operator = new Operator();
        InputParser inputParser = new InputParser(input, operator);
        String[] parsedInput = inputParser.parseInput();

        return of(parsedInput, operator);
    }

    int calculate() {
        int result = Integer.parseInt(parsedInput[0]);

        for (int idx = 1; idx < parsedInput.length; idx += 2)
            result = operator.calculateWithOperator(result, parsedInput[idx], Integer.parseInt(parsedInput[idx + 1]));

        return result;
    }
```
`of` method를 적절하게 사용해주셨네요!

- 메소드는 카멜케이스로 작성해주세요!
- of를 하나 더 만들어서 사용하면 더 좋을 것 같아요~ 이러면 생서자를 private으로 만들 수 있습니다. 

다음 스텝 진행하실 때 참고해주시면 될 것 같습니다.